### ISSUE TITLE
Homebrew 10.17.4 upgrade issue

### ISSUE THREAD

**Issue by chenrui333** [balena-io/balena-cli#1288]
Relates to https://github.com/Homebrew/homebrew-core/pull/40670
Build ref: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/43164/version=mojave/console

Local debug log:
```
$ DEBUG=1 /usr/local/Cellar/balena-cli/10.17.4/bin/balena login --credentials --email johndoe@gmail.com --password secret
ReferenceError: window is not defined
    at intlNumberFormatFormatter (/usr/local/Cellar/balena-cli/10.17.4/libexec/lib/node_modules/balena-cli/node_modules/moment-duration-format/lib/moment-duration-format.js:1723:9)
    at featureTestFormatter (/usr/local/Cellar/balena-cli/10.17.4/libexec/lib/node_modules/balena-cli/node_modules/moment-duration-format/lib/moment-duration-format.js:684:28)
    at /usr/local/Cellar/balena-cli/10.17.4/libexec/lib/node_modules/balena-cli/node_modules/moment-duration-format/lib/moment-duration-format.js:1728:29
    at Object.<anonymous> (/usr/local/Cellar/balena-cli/10.17.4/libexec/lib/node_modules/balena-cli/node_modules/resin-cli-visuals/build/widgets/progress.js:24:34)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function._load (/usr/local/Cellar/balena-cli/10.17.4/libexec/lib/node_modules/balena-cli/node_modules/raven/lib/instrumentation/instrumentor.js:34:35)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/Cellar/balena-cli/10.17.4/libexec/lib/node_modules/balena-cli/node_modules/resin-cli-visuals/build/visuals.js:20:13)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
```

**Comment by pdcastro** [comment]
This is actually a duplicate of issue #1281. It was fixed in CLI version 10.17.5. By the way, Homebrew is not an officially supported installation channel (it's not maintained by balena), and you may be interested in our brand new (beta) native installer for macOS, released just yesterday alongside CLI v11.0.0. Have a look at the installation instructions:
https://github.com/balena-io/balena-cli/blob/master/INSTALL.md

(The root cause of the "ReferenceError: window is not defined" was an upstream dependency bug in a npm package called `moment-duration-format`. Upstream issue: https://github.com/jsmreese/moment-duration-format/issues/133)

**Comment by pdcastro** [comment]
The native macOS installer, or the standalone zip packages, have the advantage that they include pre-built (pre-compiled) binaries, avoiding the main sources of errors that are common to the npm and homebrew installations.
I am closing this issue with the understanding that it was fixed, but feel free to reopen it as needed.
